src/hcl/lexer.py

```diff
@@ -319,12 +323,15 @@ def t_heredoc_eof(self, t):
 
     def t_COMMENT(self, t):
         r'(\#|(//)).*'
-        pass
+        t.value = t.value.lstrip('#').lstrip('//').lstrip()
```

Comment:
This should be moved inside the 'if' statement

Reply:
Right! Done!